repository-data/webfiles/src/main/resources/site/freemarker/intranet/task.ftl

```diff
@@ -131,71 +143,76 @@
-                <@hst.setBundle basename="intranet.headers, intranet.labels, intranet.task" />
+                    <#-- Restore the bundle -->
+                    <@hst.setBundle basename="intranet.headers, intranet.labels, intranet.task" />
 
-                <#if hasResponsibleTeams>
-                    <div id="${slugify(responsibleTeamsSectionHeader)}" class="article-section no-border">
-                        <h2>${responsibleTeamsSectionHeader}</h2>
-                        <ul>
-                            <#list document.responsibleTeams as team>
-                                <li><a href="<@hst.link hippobean=team />">${team.title}</a></li>
-                            </#list>
-                        </ul>
-                    </div>
-                </#if>
-
-                <div class="article-section" id="${slugify(datesSectionHeader)}">
-                    <h2>${datesSectionHeader}</h2>
-                    <div class="detail-list-grid detail-list-grid--regular">
-                        <#if document.reviewDate?has_content>
-                        <dl class="detail-list">
-                            <dt class="detail-list__key"><@fmt.message key="labels.review-date"/></dt>
-                            <dd class="detail-list__value"><@fmt.formatDate value=document.reviewDate.time?date type="date" pattern="d MMMM yyyy" timeZone="${getTimeZone()}" /></dd>
-                        </dl>
-                        </#if>
+                    <#if hasResponsibleTeams>
+                        <div id="${slugify(responsibleTeamsSectionHeader)}"
+                             class="article-section no-border">
+                            <h2>${responsibleTeamsSectionHeader}</h2>
+                            <ul>
+                                <#list document.responsibleTeams as team>
+                                    <li>
+                                        <a href="<@hst.link hippobean=team />">${team.title}</a>
+                                    </li>
+                                </#list>
+                            </ul>
+                        </div>
+                    </#if>
 
-                        <dl class="detail-list">
-                            <dt class="detail-list__key"><@fmt.message key="labels.last-modified"/></dt>
-                            <dd class="detail-list__value">
-                                <@fmt.formatDate value=document.lastModified?date type="date" pattern="d MMMM yyyy h:mm a" timeZone="${getTimeZone()}" var="lastModifiedDate" />
-                                ${lastModifiedDate?replace("AM", "am")?replace("PM", "pm")}
-                            </dd>
-                        </dl>
+                    <div class="article-section"
+                         id="${slugify(datesSectionHeader)}">
+                        <h2>${datesSectionHeader}</h2>
+                        <div class="detail-list-grid detail-list-grid--regular">
```

Comment:
This section needs to be update as it's using old styling. I think you may want to style this as a summary list?

https://design-system.digital.nhs.uk/storybook/latest/?path=/docs/design-system-components-atoms-summary-list--lab-component

Reply:
done